www/src/components/unbird.js

```diff
@@ -0,0 +1,240 @@
+import React from "react"
+import PropTypes from "prop-types"
+import axios from "axios"
+import styled from "react-emotion"
+import { rhythm, options } from "../utils/typography"
+import { colors } from "../utils/presets"
+import EnvelopeIcon from "react-icons/lib/fa/envelope-o"
+import CancelIcon from "react-icons/lib/md/close"
+import SendIcon from "react-icons/lib/io/paper-airplane"
+
+const FeedbackComponent = styled(`section`)`
+  box-sizing: border-box;
+  position: relative;
+`
+
+const FeedbackToggle = styled(`div`)`
+  width: 60px;
+  height: 60px;
+  background-color: ${colors.gatsby};
+  color: #fff;
+  border-radius: 100%;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.06), 0 2px 32px rgba(0, 0, 0, 0.16);
+  position: fixed;
+  bottom: 30px;
+  right: 30px;
+  z-index: 99999;
+  background-size: 30px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${colors.gatsbyDarker};
+  }
+`
+
+const IconWrapper = styled(`div`)`
+  position: relative;
+  width: 60px;
+  height: 60px;
+`
+
+const StatusMessage = styled(`span`)`
+  position: absolute;
+  width: 100%;
+  background: ${colors.gray.dark};
+  bottom: 60px;
+  color: #fff;
+  font-size: 16px;
+  padding: 0.4rem 0.8rem;
+  text-align: left;
+`
+
+const FeedbackForm = styled(`div`)`
+  position: fixed;
+  right: 30px;
+  bottom: 100px;
+  width: 350px;
+  height: 300px;
+  background-color: ${colors.gatsby};
+  box-shadow: 0 0 40px 5px rgba(0, 0, 0, 0.2);
+  border-radius: 3px;
+  overflow: hidden;
+  flex-direction: column;
+  justify-content: space-between;
+  text-align: center;
+  font-family: ${options.systemFontFamily.join(`,`)};
+`
+
+const Label = styled(`label`)`
+  width: 100%;
+  height: 240px;
+  background-color: ${colors.gatsby};
+  color: #fff;
+  font-weight: 200;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 40px;
+  font-size: 22px;
+`
+
+const Form = styled(`form`)`
+  height: 100%;
+  position: relative;
+`
+
+const Input = styled(`input`)`
+  height: 60px;
+  width: 100%;
+  font-size: 14px;
+  padding: 20px 60px 20px 20px;
+  border: none;
+  resize: none;
+
+  &:hover {
```

Comment:
Removing `outline` without any substitution is accessibility flaw. I assume that you do not wont to show outline because the the `Send` button is placed above the input. I suggest using using `float: left;` for all form elements so they will just fit one to another

```
const Label = styled(`label`)`
  font-family: ${options.headerFontFamily.join(`,`)};
  font-weight: 600;
  height: 240px;
  color: #fff;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 40px;
  font-size: 22px;
  float: left;
`

const Form = styled(`form`)`
  margin: 0;
`

const Input = styled(`input`)`
  float: left;
  height: 60px;
  width: calc(100% - 60px);
  font-size: 14px;
  padding: 20px;
  border: none;
  resize: none;
  border-right: 1px solid #ddd;
  border-radius: 0;
`

const Send = styled(`button`)`
  float: left;
  width: 60px;
  height: 60px;
  cursor: pointer;
  border: none;
  background: #fff;
  padding: 0;

  svg {
    width: 50%;
    height: auto;
    fill: ${colors.gatsby};
  }
`
```

I also removed here the inline `Send` styles and used the same svg positioning as for `Toggle`.

Reply:
The outline removal was a hang-on from porting over the widget, I think. Great catch, thank you.